packages/rabbitmq-server/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rabbitmq/rabbitmq-server
+TERMUX_PKG_DESCRIPTION="Feature rich, multi-protocol messaging and streaming broker"
+TERMUX_PKG_LICENSE="MPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="4.0.3"
+TERMUX_PKG_SRCURL=https://github.com/rabbitmq/rabbitmq-server/releases/download/v${TERMUX_PKG_VERSION}/rabbitmq-server-generic-unix-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=607ba3b4fd1fc133f3bada1b2225ff0527f99cc6eb32423ae73f6c9f5ee9bd13
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="erlang"
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_SERVICE_SCRIPT=("rabbitmq-server"
```

Comment:
`TERMUX_PKG_SERVICE_SCRIPT` splits off 2 fields from the array.
https://github.com/termux/termux-packages/blob/da5dac7f36383188e1c5b63a67ca1427c101763a/scripts/build/termux_step_install_service_scripts.sh#L1-L12

It may be preferable to add elements one by one to avoid confusion, especially with multi-line services.
```suggestion
TERMUX_PKG_SERVICE_SCRIPT=("rabbitmq-server")
TERMUX_PKG_SERVICE_SCRIPT+=(
"if [ -f \"$TERMUX_ANDROID_HOME/.config/rabbitmq/rabbitmq.conf\" ]; then 
	CONFIG=\"$TERMUX_ANDROID_HOME/.config/rabbitmq/rabbitmq.conf\"; else 
	CONFIG=\"$TERMUX_PREFIX/etc/rabbitmq/rabbitmq.conf\"; fi\n\
	exec rabbitmq-server \$CONFIG 2>&1"
)
```